<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWindow</class>
 <widget class="QWidget" name="PlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>361</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>Join Game as Player</string>
     </property>
     <widget class="QPushButton" name="pushButton_wholesaler">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 161, 242);</string>
      </property>
      <property name="text">
       <string>Wholesaler</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Put your Password you got from your</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_factory">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 161, 242);</string>
      </property>
      <property name="text">
       <string>Factory</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password_player">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>271</width>
        <height>25</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_retailer">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 161, 242);</string>
      </property>
      <property name="text">
       <string>Retailer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_distributor">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>230</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 161, 242);</string>
      </property>
      <property name="text">
       <string>Distributor</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Instructor here!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Select your supplier role</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>351</width>
       <height>361</height>
      </rect>
     </property>
     <property name="title">
      <string>Initial Parameters Summary</string>
     </property>
     <widget class="QTextEdit" name="textEdit_showRole_player">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>23</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 161, 242);</string>
      </property>
      <property name="overwriteMode">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_numWeeks_player">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>26</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_startInventory_player">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>12</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_backorderCost_player">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_holdingCost_player">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0.5</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_shipmentDelay_player">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_orderDelay_player">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>260</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Num of Weeks</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Starting Inventory</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Backorder Cost</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>180</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Holding Cost</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Shipment Delay</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>240</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Order Delay</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_playNow_player">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>310</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 161, 242);</string>
      </property>
      <property name="text">
       <string>Play Now</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>371</width>
       <height>361</height>
      </rect>
     </property>
     <property name="title">
      <string>Game Frame</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Week Num.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_currWeek_player">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Inventory</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Backlogs</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cumulative Cost</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Incoming Order</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Incoming Delivery</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_inventory_player">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>12</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_backlogs_player">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cumulativeCost_player">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_incomeOrder_player">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_incomeDelivery_player">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>240</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Place Your Order!</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_placeOrder_player">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>260</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_placeOrder_player">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>300</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 161, 242);</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
